src/gui/mzroll/alignmentvizwidget.cpp

```diff
@@ -254,7 +253,8 @@ vector<mzSample*> AlignmentVizWidget::getSamplesFromGroup(PeakGroup group) {
         mzSample* s = peaks[i].getSample();
         samples.push_back(s);
     }
-    sort (samples.begin(), samples.end());
+    sort (samples.begin(), samples.end(),mzSample::compSampleOrder);
+    reverse(samples.begin(),samples.end());
```

Comment:
Why reverse? Mention in comments maybe.

Reply:
It's making ordering of sample in Barplot and Ligand widget more consistent- both will have same color from upper row to lower row.

Reply:
mention this in comments. This is not obvious.